test/tools/build-push-e2e-catalog.sh

```diff
@@ -7,59 +7,61 @@ set -o pipefail
 help="
 build-push-e2e-catalog.sh is a script to build and push the e2e catalog image using kaniko.
 Usage:
-  build-push-e2e-catalog.sh [NAMESPACE] [TAG]
+  build-push-e2e-catalog.sh [NAMESPACE] [IMAGE] [TAG]
 
 Argument Descriptions:
   - NAMESPACE is the namespace the kaniko Job should be created in
+  - IMAGE is the full image name to build and push the catalog image
   - TAG is the full tag used to build and push the catalog image
 "
 
-if [[ "$#" -ne 2 ]]; then
+if [[ "$#" -ne 3 ]]; then
   echo "Illegal number of arguments passed"
   echo "${help}"
   exit 1
 fi
 
 namespace=$1
-tag=$2
+image=$2
+tag=$3
```

Comment:
Instead of passing an extra `tag` argument, can we parse the `image` argument? That would decrease chance of mistakes and copy/pasta errors.

Reply:
good call. I will update it.